pkg/sfu/forwarder.go

```diff
@@ -191,7 +191,7 @@ type ForwarderState struct {
 	ReferenceLayerSpatial int32
 	PreStartTime          time.Time
 	ExtFirstTS            uint64
-	RefTSOffset           uint64
+	DummyStartTSOffset    uint64
```

Comment:
Re-naming this as it is used when dummy start is done to trigger Pion's OnTrack.